### ISSUE TITLE
Moving lots of small files is too slow

### ISSUE THREAD

**Issue by Derp0** [Zarh/ManaGunZ#80]
Hi,

Transferring several thousand small files takes hours to complete. They all total to about 60MB. Also if the process is cancelled midway it just freezes (The drive's indicator keeps flashing though).
Any way to make it faster?

Using the latest pre-release version and the drive is in FAT32 format.

**Comment by Zarh** [comment]
I think, it can be faster with async copy... but this is possible only with native supported hdd (fat32 and internal)...
I don't think I'm going to work on this soon because I already made (lot of) attempts to increase the copy speed and I never succeed.
I keep it open maybe one day I'll have a revelation :)

**Comment by Derp0** [comment]
Fair enough. But what about the freezing after the cancellation? Isn't that a bug? Or is it due to the slow speed?
Hope you do :D

**Comment by Zarh** [comment]
I don't think the freeze is related to the speed, i'll try to fix it.
Was it with the filemanager ? How did you make the copy ? Did you select a directory with lot of file inside or maybe you selected each file ? how many approximatively ? 2000+ ?
Extern (fat32) to intern ?

**Comment by Derp0** [comment]
Using Managunz latest pre-release. I copied a directory with ~4000 small files inside to an external FAT32 flash drive.

**Comment by Zarh** [comment]
I just tested it and seems to work without freeze.
Please, can you test with the lastest beta version (from 20 November, 2021) : https://github.com/Zarh/ManaGunZ/releases/download/1.41/ManaGunZ_v1.41_2011.pkg. I probably fixed it when I fixed something else, or maybe I just forgot :)

**Comment by Derp0** [comment]
OK, so it seems the problem was that after the cancellation it was trying to delete the copied files and because it was too slow it would appear to be frozen. Tested the beta, and there seems to be a bit of transfer speed increase as well, thanks.
But why does it keep looping the loading indicator in the upper left under Settings?
Here's a screenshot if you don't know what I mean:
https://i.imgur.com/aWm3ZZx.png

BTW, what's the purpose of this Settings menu when there's a universal button for it?

**Comment by Zarh** [comment]
Oh, I think you are right maybe it's deletion process that made you think it's freezing. I'll remove it from the thread used to display the loading. It was a bad idea, especially when there is lot of object to delete.

This bar is used when it load PIC1 (background), it shouldn't load any picture when you are in this column, it's weird. I'll try to fix it.

You are right about the column settings, it's absolutely useless, i'll remove it. 
I wanted to display the settings in this column but nothing is done yet. So, until it's done i'll just remove it.

**Comment by Derp0** [comment]
Yeah, that makes sense. Thanks for the explanation.

I also just found another bug with this beta release. If I enter and exit about 41 directories (or enter and exit one directory about 41 times consecutively) it freezes and I have to long press the power button on my PS3 to turn it off because restarting (Via Webman mod combo) does not work. BTW, I use L1+Left/Right Dpad to navigate. I have not tested the three dot placeholder to see whether it's the same or not.
